docs/api/openapi3.json

```diff
@@ -3176,8 +3176,11 @@
                     "format": "email",
                     "type": "string"
                   },
-                  "setOneTimePassword": {
-                    "type": "boolean"
```

Comment:
this setting was superfluous. we know whether to generate one based on provider, and there's no harm setting a temp password that is never used.